GSoC-Participants.md

```diff
@@ -0,0 +1,135 @@
+---
+layout: default
+title: GSoC participants
+---
+
+This document collects the list of contributors who've contributed
+to Git via GSoC.
+
+<!-- [ [project]() ] [ [final report]() ] [ [blog]() ] -->
+
+### 2023
+
+*TODO*
+
+#### References
+
+- [GSoC archive](https://summerofcode.withgoogle.com/archive/2023/organizations/git)
+
+### 2022
+
+*TODO*
+
+#### References
+
+- [GSoC archive](https://summerofcode.withgoogle.com/archive/2022/organizations/git)
+
+### 2021
+
+*TODO*
+
+#### References
+
+- [GSoC archive](https://summerofcode.withgoogle.com/archive/2021/organizations/6398200235163648)
+
+### 2020
+
+*TODO*
+
+#### References
+
+- [GSoC archive](https://summerofcode.withgoogle.com/archive/2020/organizations/5445576591671296)
```

Comment:
From _posts/2020-05-28-edition-63.markdown:

  - Abhishek Kumar will work on
    [Implement Generation Number v2](https://summerofcode.withgoogle.com/projects/#6140278689234944).
    He will be co-mentored by Jakub Narebski and Derrick Stolee.
    More information is available on his
    [proposal](https://lore.kernel.org/git/20200326101520.GA19326@Abhishek-Arch/)
    and [blog](https://abhishekkumar2718.github.io/blog).

  - Hariom Verma  will work on
    [Unify ref-filter formats with other \-\-pretty formats](https://summerofcode.withgoogle.com/projects/#4593212745842688).
    He  will be co-mentored by Heba Waly and Christian Couder.
    More information is available on his
    [proposal](https://lore.kernel.org/git/CA+CkUQ966swTrR7D2vxgQ2ZA3E=Le=u8yvEAopOsphoCWGgDeg@mail.gmail.com/)
    and [blog](https://harry-hov.github.io/blogs/posts/).

  - Shourya Shukla will work on
    [Convert submodule to builtin](https://summerofcode.withgoogle.com/projects/#6451304047575040).
    He will be co-mentored by Kaartic Sivaraam and Christian Couder.
    More information is available on his
    [proposal](https://lore.kernel.org/git/20200325185053.10274-1-shouryashukla.oo@gmail.com/)
    and [blog](https://shouryashukla.blogspot.com/).